app/assets/stylesheets/_fonts.styl

```diff
@@ -19,4 +19,7 @@
       color zircon
 
 .red
-  color warning
\ No newline at end of file
+  color warning
+
+.delete-course-prerequisite
+  margin 0px 24px 12px 24px !important;
```

Comment:
Why is this in `_fonts`?

Can it be implemented without using `!important` (for example by using a more specific selector based on the classes of parent elements of the one you are styling)?